docs/source/topics/backends/snmp.rst

```diff
@@ -0,0 +1,149 @@
+|            |   - buffered: buffered memory in bytes                                                                     |
+|            |   - free: free memory in bytes                                                                             |
+|            |   - cached: cached memory in bytes                                                                         |
+|            | - swap:                                                                                                    |
+|            |   - total: total swap storage in bytes                                                                     |
+|            |   - free: free swap storage in bytes                                                                       |
++------------+------------------------------------------------------------------------------------------------------------+
+| interfaces | Each interface is listed with the following information:                                                   |
+|            |                                                                                                            |
+|            | - name                                                                                                     |
+|            | - type                                                                                                     |
+|            | - statistics:                                                                                              |
+|            |                                                                                                            |
+|            |  - rx_bytes                                                                                                |
+|            |  - tx_bytes                                                                                                |
++------------+------------------------------------------------------------------------------------------------------------+
+
+AirOS example
+=============
+
+::
+
+ from netengine.backends.snmp import AirOS
+ device = AirOS("10.40.0.130")
+ device.name
+ 'RM5PomeziaSNode'
+ device.uptime_tuple
+ (121, 0, 5)  # a tuple containing device uptime hours, mins and seconds
+
+We have just called two simple properties on **device**, but we can ask **device** for more specific values or portions of the SNMP tree not included in the API, just type::
+ device.next("1.3.6")
+
+Otherwise, if you want simply a value of the tree just type::
+ device.get_value("oid_you_want_to_ask_for")
+
+To collect the whole json::
+ device.to_json()
+
+*******
+OpenWRT
+*******
+
+With OpenWRT, Netengine is able to collect the following information which is returned in the
+`NetJSON Devicemonitoring <https://netjson.org/rfc.html#name-devicemonitoring>`_ format:
+
++------------+----------------------------------------------------------+
```

Comment:
please let's switch from table sytnax which is painful to maintain to simple headings and list after headings.